tuf/keys.py

```diff
@@ -223,6 +233,9 @@ def generate_rsa_key(bits=_DEFAULT_RSA_KEY_BITS):
   if _RSA_CRYPTO_LIBRARY == 'pycrypto':
     public, private = tuf.pycrypto_keys.generate_rsa_public_and_private(bits)
   
+  elif _RSA_CRYPTO_LIBRARY == 'pyca-cryptography':
```

Comment:
This idiom (testing for things in `_RSA_CRYPTO_LIBRARY` or the ED25519 counterpart) is so common in this module that my suggestion would be to consider to refactor it along the lines of
```
try:
  import one_rsa_lib as rsa_lib
except ImportError:
  pass
try:
  import another_rsa_lib as rsa_lib
....
```

Also saves you tons of identically set up checks later on when you go and check for specific implemtations to be availble (and then call identically-named functions from them).